app/scripts/modules/google/src/loadBalancer/configure/http/commandBuilder.service.js

```diff
@@ -44,52 +47,83 @@ module(GOOGLE_LOADBALANCER_CONFIGURE_HTTP_COMMANDBUILDER_SERVICE, [
     gceHttpLoadBalancerTransformer,
     loadBalancerReader,
     gceAddressReader,
+    gceXpnNamingService,
   ) {
-    function buildCommand({ originalLoadBalancer, isNew }) {
+    function buildCommand({ originalLoadBalancer, isNew, isInternal }) {
```

Comment:
Do we need to be passing around an `isInternal` boolean? I'm not very familiar with this code, but would prefer we find another way since none of the other load balancer types required an additional parameter to these methods.

Reply:
This is difficult problem.
I believe this way is better than others(e.g. copy and pasting)...
Do you have a better idea?

Reply:
Definitely prefer this approach to copy/pasting! I didn't realize until now that this service was specific to HTTP LBs. I thought this was general to all LBs, but in the context of just HTTP LBs, the `isInternal` boolean makes a lot more sense! Sorry about that!